src/libraries/System.Console/tests/ReadKey.cs

```diff
@@ -39,19 +39,33 @@ public static void ConsoleKeyValueCheck()
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser)]
     public void NumberLock_GetUnix_ThrowsPlatformNotSupportedException()
     {
         Assert.Throws<PlatformNotSupportedException>(() => Console.NumberLock);
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public void NumberLock_Browser()
+    {
+        Assert.False(Console.NumberLock);
```

Comment:
this actually sounds like something we should fix in our Wasm Console implementation. If other Unix implementations already throw PNSE from Console.NumberLock we should do the same on Wasm.

Reply:
We have `public static bool NumberLock => false;` for both [Android ](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Console/src/System/ConsolePal.Android.cs#L50)and [iOS ](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Console/src/System/ConsolePal.iOS.cs#L55)implementations.

Reply:
Ok, we should change these implementations then to be aligned with the other Unix implementations. We can do that in a follow-up PR.